install/install.go

```diff
@@ -636,6 +714,159 @@ func K8sInstaller(c *k8s.Client, o Options) error {
 	return nil
 }
 
+func actionConfigInit(ns string, settings *cli.EnvSettings) *action.Configuration {
+	actionConfig := new(action.Configuration)
+	if err := actionConfig.Init(settings.RESTClientGetter(), ns, os.Getenv("HELM_DRIVER"), log.Printf); err != nil {
+		fmt.Println("failed to initialize Helm configuration: " + err.Error())
+		return nil
+	}
+	return actionConfig
+}
+
+// K8sInstaller using operator for karmor
+func K8sInstaller(c *k8s.Client, o Options) error {
+	ns := o.Namespace
+	releaseName := "kubearmor-operator"
+
+	settings := cli.New()
+	settings.Debug = true
+	settings.SetNamespace(ns)
+
+	var printYAML []interface{}
+
+	kubearmorConfig := getOperatorCR(o)
+
+	if o.Save {
+		printYAML = append(printYAML, kubearmorConfig)
+
+		currDir, err := os.Getwd()
+		if err != nil {
+			return err
```

Comment:
o.Save is meant to help with a dry run of what the CLI is going to install.

It's meant to be same as https://helm.sh/docs/helm/helm_template/

Just saving the CR is not enough, we need to generate the entire manifest with the operator YAML as well.